src/client/game.cpp

```diff
@@ -3099,6 +3100,8 @@ void Game::processPlayerInteraction(f32 dtime, bool show_hud)
 			!runData.btn_down_for_dig,
 			camera_offset);
 
+	player->getOffhandWieldedItem(&offhand_item, &use_item, itemdef_manager, pointed);
```

Comment:
Looking at this I see a limitation that should be documented:
The offhand item cannot have its own range or liquids_pointable.

Reply:
Added to docs.